spec/features/idv/doc_auth/verify_step_spec.rb

```diff
@@ -152,7 +152,7 @@
     expect(fake_analytics).to have_logged_event(
       Analytics::THROTTLER_RATE_LIMIT_TRIGGERED,
       throttle_type: :idv_resolution,
-      step_name: Idv::Steps::VerifyWaitStepShow.class_name,
+      step_name: Idv::Steps::VerifyWaitStepShow,
```

Comment:
let's make this expect the string?
```suggestion
      step_name: 'Idv::Steps::VerifyWaitStepShow',
```